apps/src/lab2/views/Lab2Wrapper.module.scss

```diff
@@ -92,17 +93,7 @@
   }
 
   .pageErrorContainer {
-    background-color: rgba(0 0 0 / 0.5);
-    color: $neutral_dark;
-    position: absolute;
-    width: 100%;
-    height: 100%;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    top: 0;
-    left: 0;
-    z-index: 80;
+    @extend %fullscreen-overlay;
```

Comment:
Nice!  

Tiny nit: it's not quite "fullscreen" but like "full lab" or "full screen except for the header". :)

Reply:
Ah fair, how about `lab-overlay` or `full-lab-overlay`?

Reply:
Actually I think i'll do `full-container-overlay` since the overlay itself is 100% width/height with a `top`/`left` of 0, so it depends on the container it's applied to whether it covers the full screen or not.